common/scripts/base.sh

```diff
@@ -99,8 +99,7 @@ function undeploy() {
 # $2 - service port
 function waitUntilServiceIsActive() {
   marathon_lb_host_ip=$(dcos marathon app show marathon-lb | $mesos_artifacts_home/common/scripts/get-marathon-lb-host.py)
```

Comment:
We are checking the service ports now. Yes, we can check for host ports for dbs since we are using host ports now in dbs. Since we are using service ports in wso2 product application, we have to check the service via service port. Is that correct?